bookkeeper-server/src/main/java/org/apache/bookkeeper/client/SyncCallbackUtils.java

```diff
@@ -17,12 +17,11 @@
  */
 package org.apache.bookkeeper.client;
 
-import com.google.common.collect.Iterators;
+import static org.apache.bookkeeper.client.LedgerHandle.LOG;
```

Comment:
this is not needed.

Reply:
Thanks. It is originally there, this change adjust the order of importing, will handle it.